File: internal/content/testdata/bench/java/split-odd-even-indices.java
class Solution {
    public String[] split_odd_even(String s) {
        StringBuilder even = new StringBuilder();
        StringBuilder odd = new StringBuilder();
        for (int i = 0; i < s.length(); i++) {
            if (i % 2 == 0) even.append(s.charAt(i));
            else odd.append(s.charAt(i));
        }
        return new String[]{even.toString(), odd.toString()};
    }
}

File: internal/content/testdata/bench/javascript/split-odd-even-indices.js
function split_odd_even(s) {
    let even = '';
    let odd = '';
    for (let i = 0; i < s.length; i++) {
        if (i % 2 === 0) even += s[i];
        else odd += s[i];
    }
    return [even, odd];
}
